javascriptv3/example_code/dynamodb/README.md

```diff
@@ -44,34 +49,32 @@ This is a workspace where you can find the following AWS SDK for JavaScript vers
 - [Update an item in a table using PartiQL](src/partiQL_examples/src/partiql_updateItem.js) (ExecuteStatementCommand)
 - [Update items in a table by batch using PartiQL](src/partiQL_examples/src/partiql_batch_updateItems.js) (BatchExecuteStatementCommand)
 
-**Note**: All code examples are written in ECMAscript 6 (ES6). For guidelines on converting to CommonJS, see 
-[JavaScript ES6/CommonJS syntax](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/sdk-examples-javascript-syntax.html).
 
-## Getting started
+### Scenario 
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
+
+- [DynamoDB basics](scenarios/dynamodb_basics/src/dynamodb_basics.js)
+- [PartiQL basics](scenarios/partiQL_basics/src/partiQL_basics.js)
+- [PartiQL batch_basics](scenarios/partiQL_basics/src/partiQL_batch_basics.js)
+
+## Run the examples
 
-1. Clone the [AWS Code Samples repo](https://github.com/awsdocs/aws-doc-sdk-examples) to your local environment. 
-See [the Github documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) for 
-instructions.
+### Prerequisites
 
-2. Install the dependencies listed in the package.json.
+- [Set up AWS SDK for JavaScript](../README.md#prerequisites)
+
+### Steps
+
+1. Install the dependencies.
 
-**Note**: These include the client modules for the AWS services required in these example, 
-which are *@aws-sdk/client-dynamodb*, *@aws-sdk/lib-dynamodb*.
 ```
-npm install node -g
 cd javascriptv3/example_code/dynamodb
 npm install
 ```
-3. In your text editor, update user variables specified in the ```Inputs``` section of the sample file.
-
-4. Run sample code:
-```
-cd src
-node [example name].js // For example, node ddb_batchgetitem.js
-```
+3. Follow the instructions at the top of the example.
 
 ## Unit tests
-For more information see, the [README](../README.rst).
+For more information see, the [README](../README.md).
```

Comment:
For more information see, the --> For more information, see the

Reply:
done